inc/saiswitch.h

```diff
@@ -3048,6 +3048,42 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_MAX_ICMP_ECHO_SESSION,
 
+    /**
+     * @brief List of TCs eligible for trimming
+     *
+     * @type sai_u8_list_t
+     * @flags CREATE_AND_SET
+     * @default empty
+     */
+    SAI_SWITCH_ATTR_PACKET_TRIMMING_ENABLE_TC_LIST,
+
+    /**
+     * @brief Packet trimming size
+     *
+     * @type sai_uint32_t
+     * @flags CREATE_AND_SET
+     * @default 0
+     */
+    SAI_SWITCH_ATTR_PACKET_TRIMMING_SIZE,
+
+    /**
+     * @brief New packet trimming DSCP value
```

Comment:
Let's clarify that the trimmed packet can have its DSCP re-written. If this attribute is set the configured value shall be used as a new DSCP for packet after trimming . If this attribute is not set or set to 0 (need or the attribute just don't need to be set) no DSCP re-write is done. Let's support the full flexibility. Agree ?